cli/src/command_error.rs

```diff
@@ -327,7 +327,9 @@ impl From<SnapshotError> for CommandError {
   - Run `jj config set --repo snapshot.max-new-file-size {}`
     This will increase the maximum file size allowed for new files, in this repository only.
   - Run `jj --config-toml 'snapshot.max-new-file-size={}' st`
-    This will increase the maximum file size allowed for new files, for this command only.",
+    This will increase the maximum file size allowed for new files, for this command only.
```

Comment:
Perhaps the distinction should be made that with `--config-toml`, the file will likely get snapshotted and not warn anymore, while with `--ignore-working-copy`, it will not. Something like:

> - Run `jj --config-toml 'snapshot.max-new-file-size={}' st`
>   This will increase the maximum file size allowed for new files, for this command
> only. Any such command will snapshot all the files under the higher limit. This
> warning should not appear again until other large files are added to the repository.
>  - Run a one-off informational command with `--ignore-working-copy` (e.g. `jj log --ignore-working-copy`).
>    This will ignore changes to the working copy. No files will be snapshotted and this warning will occur again on the next command.